### DISCUSSION TITLE
Issues with Thunderbird 91 beta

### DISCUSSION THREAD

**Discussion by Standard8** [protz/thunderbird-conversations#1687] [General]
This thread is for discussion of issues with Thunderbird 91 beta.

In issue #1686 I have listed the known issues when running Conversations with the latest Thunderbird 91 beta.

If you think there are other issues specific to Thunderbird 91 beta, please discuss them against this issue in sub-threads.

You can also watch for updates on the issue by visiting it and hitting subscribe.

**Comment by ikirudennis** [comment]
It seems the Delete button is no longer working in 91 beta. I haven't tested the spam button, but everything else seems to operate as intended.

edit: I suppose I should mention this is on Linux, in case that might be relevant

**Reply by Standard8** [reply]
Thank you for finding this, I've just done a fix in the repository, will release that in the next update.

**Reply by ikirudennis** [reply]
Many thanks!

**Reply by Standard8** [reply]
I've just posted 3.2.29 on addons.mozilla.org for approval with the delete fix, this should go out in the next few days.

**Comment by ikirudennis** [comment]
Conversation view doesn't respect dark mode theme. It's all blindingly white. Again, this is on Linux, in case that's relevant.

**Reply by Standard8** [reply]
Dark theme has never worked properly and is an [existing project](https://github.com/thunderbird-conversations/thunderbird-conversations/projects/5) which I'd like to do (or have some help with), but haven't got around to yet - improving the add-on for the future and getting quick reply working are the higher priorities.

**Reply by ikirudennis** [reply]
Fair enough. Thanks for the update!

**Comment by ikirudennis** [comment]
Clicking on a link in an email doesn't open the link. Right-clicking and selecting "Open Link in Browser" _does_ work, however.

**Reply by mitra42** [reply]
I'm seeing the same problem with links - and right click / open link in browser also works for me.

**Reply by Standard8** [reply]
Yeah, I'd seen that before and forgotten about it. I'll add it to the list - I think I probably know what's caused it, will investigate later in the week.

**Reply by mitra42** [reply]
At least some links have started (almost) working now - though my version is still 3.2.28 from 26 July and TB itself is still 91.0b3.   The problem now is that they are opening in Thunderbird, rather than in the Browser. 

If I switch to Classic Reader the links open correctly (in Firefox in my case) 

I presumed that must be a setting that Conversations is ignoring as it works correctly in Classic, but i can't find that setting. Though https://support.mozilla.org/en-US/kb/hyperlinks-in-messages-not-working#w_check-that-no-application-is-specified-for-the-http-content-type suggests its the Files & Attachments - having no setting for HTTP means to go to the default browser. 

This is important to me because when links open in TB they also don't get the saved passwords that are in Firefox, so for example linkedin and facebook links don't open correctly.

**Reply by Standard8** [reply]
I've just fixed the links issue in 3.2.29 (awaiting approval on AMO). It wasn't a setting - just to do with where Conversations needed to hook into handling click events.

**Reply by mitra42** [reply]
Thanks Mark, was it the same problem - some links not opening, and others opening in local browser ?

**Comment by amourchaleur** [comment]
In the conversation view, printing doesn't work. Not able to print preview and nothing shows up when we print. Current workabout is to view using classic reader and print from there.

**Reply by Standard8** [reply]
Thank you for mentioning this, I see it as well. I'll take a look and see what's happening.

**Reply by amourchaleur** [reply]
Awesome, thanks! Oh and if we search for messages, in the search results, if we mouse-over any of the contacts, the "send email" icon doesn't work (ie. after I click on it, it does not open up a compose window for that contact). Appreciate if you can look into that as well. Thank you!

**Reply by Standard8** [reply]
I've just finished fixing the print issue and the contact send email one.

For the printing you'll be able to print if a single message is selected. You can't print multiple single messages at the moment - Thunderbird has currently disabled that because the new printing system doesn't support it in the same way.

I do hope to bring back printing a conversation soon, I think I now know what I can do to fix that.

I've just posted a new version with the fixes for approval, it might be a couple of days before it gets reviewed.

**Reply by amourchaleur** [reply]
Thank you so much! For the conversation printing, if the conversation has only 1 message and even if that 1 message is selected, I can't get the print preview to work as well. Nothing pop ups after we click on the print. But if there are more than 1 messages in that conversation and one of the messages are selected, the print is working. 

Confirmed that the contact send email is working already, thanks!

**Comment by amourchaleur** [comment]
Quick question, is there a way to show the full subject of the email at the top of the conversation view? Thanks!

**Reply by Standard8** [reply]
The full subject of the conversation should be displayed if there is space, if not there should be a tooltip.

If you mean of the message itself, then that's #192

**Comment by rpstlv** [comment]
After updating to TB91* attachements no longer show up with Conversations 4.0.0 enabled. One needs to use e.g. "classic reader" in order to see them.

**Reply by wsmwk** [reply]
@rapostolo this thread is about Thunderbird 91 BETA which is long ago finished. You must be using the current 91 RELEASE. 
Regardless, I see no open attachment problems reported against Conversations https://github.com/thunderbird-conversations/thunderbird-conversations/issues?q=is%3Aissue+is%3Aopen+attachment

**Comment by Standard8** [comment]
Locking the conversation now, please start a new conversation or file an issue.
